src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,10 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+
```

Comment:
It's not good idea use _justify-content: space beatween_  because "cards should have fixed width and **fixed distances between them**" and if you change your block width space can be bigger then needed. Try use margin intead it.

Reply:
agree, let's proceed with another approach